src/utils/calculateTimeDifference.ts

```diff
@@ -0,0 +1,28 @@
+import { Timestamp } from "firebase/firestore";
+
+export const calculateTimeDifference = (timestamp: Timestamp): string => {
+  const commentDate = timestamp.toDate();
+  const now = new Date();
+  const diffInSeconds = Math.floor(
+    (now.getTime() - commentDate.getTime()) / 1000
+  );
+
+  const secondsInMinute = 60;
```

Comment:
move as constants out of the function:

const SECONDS_IN_MINUTE = 60;
const SECONDS_IN_HOUR = 3600;
const SECONDS_IN_DAY = 86400;